### ISSUE TITLE
Push notification body gets truncated 

### ISSUE THREAD

**Issue by jamilsaadeh97** [aws-amplify/amplify-android#2857]
### Before opening, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-android/issues?q=is%3Aissue+) and [discussions](https://github.com/aws-amplify/amplify-android/discussions).

### Language and Async Model

Not applicable

### Amplify Categories

Notifications

### Gradle script dependencies

Not applicable

### Environment information

Not applicable

### Describe the bug

Notifications are being truncated when the body exceeds ~90 characters. The reason is that "long" notifications are not being handled [here](https://github.com/aws-amplify/amplify-android/blob/b3c79fb1e315f253f2d5f9e5393f99ba19088ae8/aws-push-notifications-pinpoint-common/src/main/java/com/amplifyframework/pushnotifications/pinpoint/PushNotificationsUtils.kt#L130) by using the [BigTextStyle](https://developer.android.com/develop/ui/views/notifications/expanded#large-style).

I'm using Amplify Flutter and I can't override these properties in my app.

I've opened #2856 to fix this

Thank you!

### Reproduction steps (if applicable)

1. Put app in the background
2. Send a notification with body: "This is a long push notifications message that's supposed to be completely shown on the user device"

### Code Snippet

```java
// Put your code below this line.

```

### Log output

<details>

```
// Put your logs below this line


```

</details>

### GraphQL Schema

<details>

```graphql
// Put your schema below this line


```

</details>

**Comment by vincetran** [comment]
Hi @jamilsaadeh97 Thanks for the report and the PR! I'll try to get to looking at your PR before the end of the week.

**Comment by jamilsaadeh97** [comment]
Hi @vincetran looking forward for your feedback!

**Comment by mattcreaser** [comment]
Did not mean to auto-close this. We typically only close issues once the fix is released. This PR has been merged and will be included in our next release.

**Comment by mattcreaser** [comment]
This fix we released in Amplify 2.20.0 😄
